Makefile

```
@@ -58,6 +58,18 @@ test-e2e: export OASIS_INDEXER_E2E = true
 test-e2e:
 	@$(GO) test -race -coverpkg=./... -coverprofile=coverage.txt -covermode=atomic -v ./tests/e2e
 
+dump-state: oasis-indexer
+	sed -E -i='' 's/query_on_cache_miss: false/query_on_cache_miss: true/g' "tests/e2e_regression/e2e_config.yml"
+	./oasis-indexer --config tests/e2e_regression/e2e_config.yml analyze
```

```suggestion
fill-cache-for-e2e-regression: oasis-indexer
	cp tests/e2e_regression/e2e_config.yml /tmp/indexer_fill_e2e_regression_cache.yml
	sed -E -i='' 's/query_on_cache_miss: false/query_on_cache_miss: true/g' /tmp/indexer_fill_e2e_regression_cache.yml
	./oasis-indexer --config /tmp/indexer_fill_e2e_regression_cache.yml analyze
```

Two nits:
1) Renamed the target to relate to the name of the corresponding test target
2) I'd prefer to keep the use case for `tests/e2e_regression/e2e_config.yml` clear: Running the test. That's what its name+location imply, and also we'll be running the test a lot more often than we'll be filling the cache. So the suggested change above makes a temporary copy of the .yml and modifies it just for the purpose of filling the cache.